webapp/src/home/index.js

```diff
@@ -1 +1,64 @@
-export { Home } from './home-page'
+import React from 'react'
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+import { css } from '@emotion/core'
+import { Link } from 'react-router-dom'
+
+const GET_TRANSACTIONS = gql`
+  {
+    transactions {
+      id
+      amount
+      credit
+      debit
+      description
+      merchant_id
+      user_id
+    }
+  }
+`
+
+export const Home = () => {
+  const { data } = useQuery(GET_TRANSACTIONS)
+
+  const transactions = data.transactions || []
+
+  return (
+    <div css={transactionsStyle}>
```

Comment:
Question for PM: What information do we want on an overview page? Are there only a few fields a customer needs to identify a transaction, or should we show all of them?

Will the customer want to stay on the overview page generally, or do we want to push them to the single transaction page?